src/pybind/mgr/dashboard_v2/module.py

```diff
@@ -47,42 +47,42 @@ def configure_cherrypy(self, in_unittest=False):
                 'no server_addr configured; '
                 'try "ceph config-key put mgr/{}/{}/server_addr <ip>"'
                 .format(self.module_name, self.get_mgr_id()))
-        self.log.info("server_addr: %s server_port: %s", server_addr,
+        self.log.info('server_addr: %s server_port: %s', server_addr,
                       server_port)
 
         if not in_unittest:
             cherrypy.config.update({
                 'server.socket_host': server_addr,
                 'server.socket_port': int(server_port),
             })
-        cherrypy.tools.autenticate = cherrypy.Tool('before_handler',
+        cherrypy.tools.authenticate = cherrypy.Tool('before_handler',
```

Comment:
this cherrypy tool is called `autenticate` and not `authenticate`. Please revert all the changes you've made to the word `autenticate`.

Reply:
With ``cherrypy.Tool`` you create a new handler that can be named whatever you want. If there is one called ``autenticate``, then it will be overwritten with your registered tool. So it does not matter at all.

Reply:
In that case please ignore my comment. Just make sure that all tests are passing by running `tox`